MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
+sonnetz= read_data.split('\n\n')
+sonnetLine=[0, 1, 2, 3,4, 5,6 ,7, 8, 9, 10, 11, 12, 13]
+import random
+
+for k in range(0,len(sonnetz)-1):
+	lines=sonnetz[k].split('\n') #splits into lines
+
+	def DictRhymes():
+		getRhyme(0, 2) #English sonnet rhyme scheme (lines that rhyme)
+		getRhyme(1, 3)
+		getRhyme(4, 6)
+		getRhyme(5, 7)
+		getRhyme(8,10)
+		getRhyme(9,11)
+
+
+	def getRhyme(num1, num2): #num1 and num2 inputs represent which lines are selected (rhyming lines)
+		lastword1=lines[num1].split(' ')[-1] #takes the last word of every other line
+		lastword2=lines[num2].split(' ')[-1]
+		if(sentiment(lastword1)[0]<=.25 and sentiment(lastword2)[0]<=.25):
```

Comment:
Try not to use 'magic numbers' such as .25. Instead, make a variable whose name explains why the number is significant (for example, 'depressing_sentiment = .25'). This goes for pretty much any number where it's not completely obvious your intentions (like 0 or -1 are not magic numbers). An exception would be when you call 'getRhyme' 6 times--you probably shouldn't make variables for all those line numbers. However, you should probably comment there why those numbers you chose are significant (ie. A sonnet rhyme scheme always rhymes these specific line numbers)